app/controllers/hyrax/admin/workflow_roles_controller.rb

```diff
@@ -0,0 +1,41 @@
+module Hyrax
+  module Admin
+    class WorkflowRolesController < ApplicationController
```

Comment:
This is better, but I was suggesting getting rid of this controller and adding the additions in an initializer.  That results in less copy/paste code that we have to hunt down later.